models/workitemtype_repository_blackbox_test.go

```diff
@@ -0,0 +1,50 @@
+package models_test
+
+import (
+	"golang.org/x/net/context"
+
+	"testing"
+
+	"github.com/almighty/almighty-core/app"
+	"github.com/almighty/almighty-core/models"
+	"github.com/almighty/almighty-core/resource"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCreateLoadWIT(t *testing.T) {
+	resource.Require(t, resource.Database)
+
+	undoScript := &models.DBScript{}
+	defer undoScript.Run(db)
+	repo := models.NewUndoableWorkItemTypeRepository(models.NewWorkItemTypeRepository(db), undoScript)
+	db2 := db.Unscoped().Delete(models.WorkItemType{Name: "foo.bar"})
+
+	if db2.Error != nil {
+		t.Fatalf("Could not setup test %s", db2.Error.Error())
+		return
+	}
+
+	wit, err := repo.Create(context.Background(), nil, "foo.bar", map[string]app.FieldDefinition{
+		"foo": app.FieldDefinition{
+			Required: true,
+			Type:     &app.FieldType{Kind: string(models.KindFloat)},
+		},
+	})
+	assert.Nil(t, err)
```

Comment:
I highly recommend to switch from `assert` to `require` because `require` will abort on the first violation of an assertion which is most likely what you want anyway; otherwise you end up having many followup errors.